tests/test_workflows.py

```diff
@@ -825,16 +863,13 @@ def test_qaoa_asdict_dumps(self):
         # set the header
         qaoa.set_header(
             project_id="8353185c-b175-4eda-9628-b4e58cb0e41b", 
-            name="test", 
+            description="test", 
             run_by="raul", 
             provider="-", 
             target="-", 
             cloud="local", 
             client="-", 
-            qubit_routing="-", 
-            error_mitigation="-", 
-            error_correction="-"
```

Comment:
Double checking if these headers are planned to be removed. And is `name` to be replaced with `description`

Reply:
Yes, that is correct!